DOCS.md

```diff
@@ -309,6 +309,18 @@ The list of all understood properties is given below.
   ]);
   ```
 
+- `pileItemOffset` can be set to an array or a callback function. The array should either be a tuple specifying the x and y offset in pixel, or be empty to indicate randomly offset. E.g.,
+
+  ```javascript
+  // Align items in y-axis
+  piling.set('pileItemOffset', [0, 5]);
+
+  // Random alignment
+  piling.set('pileItemOffset', []);
```

Comment:
Let's remove this special case (in `pile.js` from line 575-620). The user should be able to implement random offset by specifying `pileItemOffset: () => [Math.random(), Math.random()]`.  This will also allow us to remove `randomOffsetRange` and `randomRotationRange`, which the user can handle. To still support random rotations, can you allow `pileItemRotation` to either be a numerical value or a function? That way the user has full control off the offset and rotation and we don't have to deal with any special cases.